docs/en/observability/aws-firehose/aws-firehose.asciidoc

```diff
@@ -0,0 +1,32 @@
+[[aws-firehose]]
+= Amazon Kinesis Data Firehose
+
+preview::[]
+
+{ecloud} users can ingest logs directly from AWS Kinesis Data Firehose.
+
+AWS Kinesis Data Firehose works with {stack} version 7.17 or greater.
+
+* <<aws-firehose-setup-guide>>
+* <<aws-firehose-troubleshooting>>
+
+[discrete]
+[[aws-firehose-what-is-it]]
+== What is Amazon Kinesis Data Firehose?
+
+Amazon Kinesis Data Firehose is an extract, transform, and load (ETL) service that reliably captures, transforms, and delivers streaming data to data lakes, data stores, and analytics services.
+
+// todo: add link to pricing page
+Using Firehose to send your data to Elastic means you have no agents to deploy, lambdas to configure, or Beats to manage. Pricing is simple, predictable, and typically more cost-effective than other data ingest solutions. Additionally, auto-scaling capability is built in, and the service is designed to handle high-volume use cases.
+
+The overall architecture is shown below.
+
+image::./images/firehose-architecture.png[Diagram showing Amazon Kinesis Data Firehose connected to Elastic Cloud with examples of input data sources, such as Amazon Kinesis Data Streams and Amazon Route 53 logs.]
+
+//[[aws-firehose-comparison]]
+//== Comparison with other ways to send data to Elastic
+
+//TODO
+
+include::aws-firehose-setup.asciidoc[leveloffset=+1]
```

Comment:
I think this OK, but you might want to remove the nesting for this to work better in a separate book. 
![image](https://user-images.githubusercontent.com/14206422/231900945-bafb0631-faf4-4093-ae26-6315624de2be.png)